mlir/lib/Dialect/SparseTensor/Transforms/SparseTensorCodegen.cpp

```diff
@@ -729,7 +730,6 @@ class SparseReMapConverter : public OpConversionPattern<ReinterpretMapOp> {
 };
 
 /// Sparse codegen rule for the alloc operator.
-/// TODO(springerm): remove when bufferization.alloc_tensor is gone
 class SparseTensorAllocConverter
     : public OpConversionPattern<bufferization::AllocTensorOp> {
```

Comment:
This seems very similar to the converter for NewOp, is it possible to merge these two?

Reply:
Sounds good (but TBD ;-)